google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/flexible/FlexibleFacetEditor.java

```diff
@@ -234,15 +234,20 @@ private void showWarnings() {
 
   private void toggleDockerfileSection() {
     boolean enabled = false;
+    boolean visible = true;
```

Comment:
not sure I follow; looks like when the exception is thrown its just ignored. I think its valid to say that if the parsing fails, then we should hide it entirely (not have it visible, but disabled)